README.md

```diff
@@ -146,19 +146,16 @@ As most datasets in HuggingFace repository should be compatible with our pipelin
 
 # Passing task model and task training parameters to predefined flair pipelines
```

Comment:
README should have a short description  (for detailed we can yous this tutorial notebook) of how to change basic and advanced configs and their purposes.  In my view, the current version is not sufficient.